clients/privacy-center/.env.development

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_AUTH_ENDPOINT=http://0.0.0.0:8080/api/v1/oauth/token
```

Comment:
It'd also be good to pass this URL down through `config.json`, so anyone hosting this has the option to connect it to a different server should they wish.

Reply:
Yes, this is the point I mentioned on our call the other day – we need to decide where we want environment variables to live for this specific project, because it has an additional config entry point in `config.json`. Either we should specify variables in `config.json` or in a `.env` file, but not both. 

My thought is that we should just remove this file – what do you think?

Reply:
Agree, I think all the config is best off in one place